prow/cluster/espv2-disk.yaml

```diff
@@ -0,0 +1,11 @@
+kind: PersistentVolumeClaim
```

Comment:
This needs to be part of the build cluster (where jobs run) not the service cluster (prow's control plane)

Reply:
You mean this line? or this whole config

Reply:
The file